### DISCUSSION TITLE
Feature suggestion: let instructors preview results as seen by student

### DISCUSSION THREAD

**Discussion by damithc** [TEAMMATES/teammates#11788] [Ideas]
Explanation: Let instructors preview session results as seen by students, before publishing results

Reason: This is a often requested feature. The reason is that instructors want to ensure students will be seeing only the responses they are intended to see

Complication: If the question has a visibility setting that restricts the the instructor's visibility (e.g., instructors are not supposed to see who submitted the response), the preview should honor those restrictions as well. In such cases.

Posting it here first as we need to discuss the feasibility and size of this feature e.g., should this be a mini-project or a micro-project?

**Comment by wkurniawan07** [comment]
Duplicate of issue #4570
